### PR TITLE
refactor: add check for the payload length in `LSP6KeyManagerCore`

### PR DESCRIPTION
**Pull request by b00ste**
### What does this PR introduce?

In this PR the following changes are made:
- A new error added, 'InvalidPayload(bytes)'.
- A check added in both internal function, `` and `` in order to verify that the payload is not smaller than 4 bytes.

### CODE REVIEW

File: tests/LSP6KeyManager/tests/PermissionCall.test.ts [hunk 1]
```diff
@@ -76,6 +76,26 @@ export const shouldBehaveLikePermissionCall = (
   });
 
   describe("when interacting via `execute(...)`", () => {
+    describe("when the payload is smaller than 4 bytes", () => {
```
Comment (CJ42) [inline tests/LSP6KeyManager/tests/PermissionCall.test.ts:79]: Could we put this test in the file `OtherScenarios.test.ts` for the Key Manager test suite?

I had already started to draft one test long time ago and left it there skipped. 

Maybe we can also re-arrange this test to ensure this new custom error is returned with an empty payload? :)

https://github.com/lukso-network/lsp-smart-contracts/blob/5f4b9a7b2e224f1536be8d5164f58b57016cafcd/tests/LSP6KeyManager/tests/OtherScenarios.test.ts#L55-L57

**Review by CJ42 — CHANGES_REQUESTED** [review]

**Review by skimaharvey — APPROVED** [review]

**Review by YamenMerhi — APPROVED** [review]
